python/pyproject.toml

```diff
@@ -15,7 +15,7 @@ python = ">=3.7.1, <4"
 whylogs-sketching = ">=3.4.1.dev3"
 protobuf = ">=3.15.5"
 importlib-metadata = { version = ">=0.23", python = "<3.8" }
-typing-extensions = ">=typing-extensions"
+# typing-extensions = ">=typing-extensions"
```

Comment:
@andyndang 👀

Reply:
Looks like the original was put in by @andyndang 5 months ago. I had readthedocs poetry fail on friday on this, but it appeared to be fixed by what @jamie256. I'm super curious what this is for?

Reply:
This is for backporting some of the language features we used in whylogs v1 that are only in python 3.10, so we only need this for older than python 3.10. Our use of TypeAlias in the decorators is one example.

Fixing the version range here: https://github.com/whylabs/whylogs/pull/842

Reply:
Ok, the pyproject.toml file is now updated in mainline with https://github.com/whylabs/whylogs/pull/845 merged.